README.md

```diff
@@ -91,17 +91,17 @@ We will backport bugfixes--but not new features--into older versions of
 
 #### Compatibility matrix
 
-|                     | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 | Kubernetes 1.8 | Kubernetes 1.9 | Kubernetes 1.10 |
-|---------------------|----------------|----------------|----------------|----------------|----------------|----------------|-----------------|
-| client-go 1.4       | ✓              | -              | -              | -              | -              | -              | -               |
-| client-go 1.5       | +              | -              | -              | -              | -              | -              | -               |
-| client-go 2.0       | +-             | ✓              | +-             | +-             | +-             | +-             | +-              |
-| client-go 3.0       | +-             | +-             | ✓              | -              | +-             | +-             | +-              |
-| client-go 4.0       | +-             | +-             | +-             | ✓              | +-             | +-             | +-              |
-| client-go 5.0       | +-             | +-             | +-             | +-             | ✓              | +-             | +-              |
-| client-go 6.0       | +-             | +-             | +-             | +-             | +-             | ✓              | +-              |
-| client-go 7.0       | +-             | +-             | +-             | +-             | +-             | +-             | ✓               |
-| client-go HEAD      | +-             | +-             | +-             | +-             | +-             | +              | +               |
+|                     | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 | Kubernetes 1.8 | Kubernetes 1.9 | Kubernetes 1.10 | Kubernetes 1.11 |
```

Comment:
Honestly, I hate this.

Can we remove 1.4 from the table? This isn't going to make it any easy for the next releases as well.

Reply:
11-4=7, that's >2 years old, right? I would drop it.